lib/puma/plugin/yabeda_prometheus.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'rack'
+require 'yabeda/prometheus/exporter'
+
+Puma::Plugin.create do
+  def start(launcher)
+    options = launcher.options
+    events = launcher.events
+
+    host = options[:prometheus_host] || ENV['PROMETHEUS_EXPORTER_BIND'] || '0.0.0.0'
+    port = options[:prometheus_port] || ENV['PROMETHEUS_EXPORTER_PORT'] || 9393
```

Comment:
Let's use 9394 for consistency with yabeda-prometheus' default (however I can't remember why it is exactly 9394): https://github.com/yabeda-rb/yabeda-prometheus/blob/4393803a273ff333c39587fe7225aee3af0c3831/lib/yabeda/prometheus/exporter.rb#L22-L26

```suggestion
    port = options[:prometheus_port] || ENV['PROMETHEUS_EXPORTER_PORT'] || 9394
```